src/Tracy/Debugger.php

```diff
@@ -400,9 +407,10 @@ public static function setLogger(ILogger $logger)
 	public static function getLogger()
 	{
 		if (!self::$logger) {
-			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen());
+			self::$logger = new Logger(self::$logDirectory, self::$email, self::getBlueScreen(), self::$fromEmail);
```

Comment:
I don't like this new property and I see no reason for the debugger to be concerned about what the is Logger doing.